style.css

```diff
@@ -122,6 +127,43 @@ button:focus {
   display: none;
 }
 
+.close-menu {
+  cursor: pointer;
+  display: none;
+}
+
+.change-menu {
+  height: 90vh;
```

Comment:
- [ ] I suggest you make this 100% to cover the screen when the menu appears for better UI(user-interface) design ✌
![screenshot (6)](https://user-images.githubusercontent.com/44624138/145980912-c714d733-5bb3-4324-adf2-da9e6253f2d8.png)